ci/deps/actions-37-locale.yaml

```
@@ -30,7 +30,7 @@ dependencies:
   - openpyxl
   - pandas-gbq
   - google-cloud-bigquery>=1.27.2 # GH 36436
-  - pyarrow>=0.17
+  - pyarrow=0.17.0 # GH 38803
```

```suggestion
  - pyarrow=0.17 # GH 38803
```

that might help for conda being able to solve it

---

Thanks! I'm more familiar with pyenv and wasn't sure about the syntax.

---

It's not that the syntax was wrong, but because of the more strict version pin, conda was apparently not able to find a solution were all version constraints were compatible. By being less strict, conda can also pick 0.17.1, and apparently, that solves the version conflicts.